main.tf

```diff
@@ -78,6 +80,64 @@ resource "aws_cloudwatch_event_bus" "this" {
     }
   }
 
+  dynamic "log_config" {
+    for_each = var.logging != null ? [var.logging] : []
```

Comment:
I think this should be its own variable `log_config` and not embedded in this `logging` variable